sendEmail.py

```diff
@@ -46,8 +47,12 @@ def sendEmail(to, roomName):
         for label in labels:
             print(label['name'])
     sender = "uninotificationserviceemail@gmail.com"
-    subject = "Room booking cancelled"
-    message_text = "The room booking for " + roomName + " was cancelled"
+    if emailType == 'booking':
+        subject = "Room booking cancelled"
+        message_text = "The room booking for " + roomName + " was cancelled"
+    else:
+        subject = "Room booking cancelled"
+        message_text = roomName + " is now available"
```

Comment:
Would it require much work to pass in the start and end times so that the user knows at which time the room is now available (they may have booked/watched the room for multiple different times)?
Also, while not essential, it would be nice if the email started with "Dear Name" or something to make it feel more personal.